packages/codegen/templates/app00/next.config.js

```diff
@@ -5,29 +5,44 @@ const withTMModule = require('next-transpile-modules')
 
 const conf = require('@open-condo/config')
 
+const { antGlobalVariables } = require('@condo/domains/common/constants/style')
+
 // Tell webpack to compile the "@open-condo/next" package, necessary
 // https://www.npmjs.com/package/next-transpile-modules
 // NOTE: FormTable require rc-table module
-// TODO(codegen): include all TypeScript modules, that you plan to use in your app, otherwise you will get errors about unregognized TypeScript syntax
+
 const withTM = withTMModule([
+    '@open-condo/codegen',
     '@open-condo/next',
+    '@open-condo/bridge',
+    '@open-condo/featureflags',
+    '@open-condo/keystone',
+    '@emotion/styled',
     '@app/condo',
     '@app/{{name}}',
+    '@condo/domains',
     '@{{name}}/domains',
     'rc-table',
 ])
 
 const serverUrl = conf['SERVER_URL']
-const apolloGraphQLUrl = `${serverUrl}/admin/api`
+const apolloGraphQLUrl = `${serverUrl}/graphql`
+const condoUrl = conf['CONDO_DOMAIN']
+const b2bAppId = conf['CONDO_B2B_APP_ID'] || null
+const defaultLocale = conf['DEFAULT_LOCALE'] || 'ru'
```

Comment:
```suggestion
const defaultLocale = DEFAULT_LOCALE
```
use `DEFAULT_LOCALE` from `constants`